### PR TITLE
Implement webauthn extra origins & Raise KeyCloak version to 24.0.5 in acceptance tests

### PR DESCRIPTION
**Pull request by TuningYourCode**
https://github.com/treydock/puppet-module-keycloak/pull/312 implements quiet a few properties on the realm resource.

The webauthn & webauthn passwordless extra origins property require at least KeyCloak 23.0.0 as it was added with that release.

This PR raises the minimum KeyCloak version used for acceptance testing and will add these properties.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -65,18 +65,18 @@ jobs:
...
-          - "22.0.0"
+          - "23.0.7"
           - "24.0.3"
         keycloak_full:
           - "no"
         include:
           - set: "el8"
             puppet: "puppet7"
-            keycloak_version: "22.0.0"
+            keycloak_version: "23.0.7"
```
Comment (treydock) [inline .github/workflows/ci.yaml:75]: These values are actually controlled by .sync.yml:

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/.sync.yml#L28

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/.sync.yml#L34

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/.sync.yml#L38

**Comment by treydock** [comment]
There are now merge conflicts due to other PR getting merged.

Also this needs to be bumped to new minimum version: 

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/spec/spec_helper_acceptance_setup.rb#L6

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/spec/classes/init_spec.rb#L11

https://github.com/treydock/puppet-module-keycloak/blob/a627fa4eb1ec58176f1260ff28a9c7833193e654/manifests/init.pp#L225

**Comment by treydock** [comment]
Actually, make 24.x the minimum as seems 25.0.0 is now released.

**Comment by treydock** [comment]
Looks like 25.0.0 may break several things.  I think it'd be fine to just upgrade this to 24.0.5 as default and only tested version and then do 25.0.0 separate which I can help with.

**Comment by TuningYourCode** [comment]
Thanks, yeah looks like 25.0.0 has a few things which need changes in the module or at least in tests.

**Comment by treydock** [comment]
I am working on 25.x support and so may not release this until I release the support for 25.x.  There are some breaking changes to support 25.x that will also require a major version bump.

**Comment by TuningYourCode** [comment]
No problem, i don't need these properties anyway - just implemented them to have all present in the module.

**Pull request merged**